dimod/discrete/discrete_quadratic_model.py

```diff
@@ -203,6 +203,94 @@ def add_linear_equality_constraint(self, terms: LinearTriplets,
         self._cydqm.add_linear_equality_constraint(
             index_terms, lagrange_multiplier, constant)
 
+    def add_linear_inequality_constraint(self, terms: LinearTriplets,
```

Comment:
Would it be desirable to implement this in the Cython DQM object, like `add_linear_equality_constraint`?

Reply:
I am not sure, the `add_linear_equality_constraint` which is already implemented in Cython does the heavy lifting in this function.